### ISSUE TITLE
The call to UseAuthorization should appear between app.UseRouting()

### ISSUE THREAD

**Issue by mpaulohs** [tesar-tech/MultipleBlazorAppsWithAuth#1]
The project is generating the following warning: The call to UseAuthorization should appear between app.UseRouting() and app.UseEndpoints(..) 
for authorization to be correctly evaluated.
![Blazor](https://user-images.githubusercontent.com/37406808/119291392-5ec4e400-bc1c-11eb-842b-b07da72d8d33.jpg)

**Comment by tesar-tech** [comment]
Yes, because we set authorization for both apps.. The second `UseAuthorization` is after first `UseEndpoints`, that is the problem I think... 

Do you have any solution to this?
